snmp/tests/test_e2e_core_profiles/test_profile__cisco_generic.py

```diff
@@ -0,0 +1,222 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import pytest
+
+from datadog_checks.dev.utils import get_metadata_metrics
+
+from .. import common
+from ..test_e2e_core_metadata import assert_device_metadata
+from .utils import (
+    assert_common_metrics,
+    assert_extend_cisco_cpu_memory,
+    assert_extend_generic_bgp4,
+    assert_extend_generic_if,
+    assert_extend_generic_ip,
+    assert_extend_generic_ospf,
+    assert_extend_generic_tcp,
+    assert_extend_generic_udp,
+    create_e2e_core_test_config,
+    get_device_ip_from_config,
+)
+
+pytestmark = [pytest.mark.e2e, common.py3_plus_only, common.snmp_integration_only]
+
+
+def test_e2e_profile__cisco_generic(dd_agent_check):
+    config = create_e2e_core_test_config('_cisco-generic')
+    aggregator = common.dd_agent_check_wrapper(dd_agent_check, config, rate=True)
```

Comment:
Had to change this test manually a lot (+ .snmprec). Mostly because of the '`cpu:X`, '`mem:X`'.